public/js/question.js

```diff
@@ -28,9 +30,9 @@ $('#mc_answerform').submit(function(evt) {
     evt.preventDefault();
     var ans = $("input[name=answer]:checked").val();
     $.ajax({
-        type: 'POST',
+        type: 'GET',
```

Comment:
This is a submission, it should stay as a POST instead of a GET. But in this case there isn't any security risk or anything, so if someone is anal about this then we can change it otherwise we can leave it.

See: https://www.tutorialspoint.com/http/http_methods.htm

Reply:
ooooh nice catch. I was using it for debugging. Will make it post again, my bad